crates/rolldown/tests/esbuild/default/require_fs_browser/artifacts.snap

```diff
@@ -21,8 +22,9 @@ source: crates/rolldown_testing/src/integration_test.rs
 
 ```js
 
+
 //#region entry.js
-console.log(require("fs"));
+console.log(__require("fs"));
```

Comment:
Here actually we are not align with esbuild, since they would emit error `fs` is not exists in `browser`, and we just treat `fs` as a external module, and rewrite `require` to `__require`